server/tests/helpers/user_helper.js

```diff
@@ -0,0 +1,21 @@
+import { fixtures, build } from '../factories';
+
+function buildUserAndGroups() {
+  return new Promise((resolve, reject) => {
```

Comment:
You don't need the new promise wrap here, the Promise.all() will return a promise.
```
 return Promise.all([b1, b2])
    .then((values) => {
      generatedValues.groups = values;
      const groupNames = values.map(g => g.name);
      return build('user', fixtures.user({ groups: groupNames }));
```